src/lib/components/prompt/PromptBar.svelte

```diff
@@ -27,7 +27,7 @@
 	}
 </script>
 
-<div class="flex h-full w-[450px] shrink-0 flex-col border-r px-6 py-4">
+<div class="flex h-full w-1/3 min-w-[300px] flex-col border-r px-6 py-4">
```

Comment:
I think it's better a bit bigger.

Reply:
I think it looks pretty weird tbh. Reverting for now. I'd rather have the option to maximize it. Now that it's wider than tall on most screens looks weird to me.